src/Func.h

```diff
@@ -1308,6 +1310,9 @@ class Func {
     /** Schedule for execution as GLSL kernel. */
     EXPORT Func &glsl(Var x, Var y, Var c);
 
+    /** Schedule for execution on Hexagon. */
```

Comment:
Way too small of a comment for something so complex. What is hexagon, and what does it mean to mark a loop as hexagon?